TreasureHunt/Assets/2.Scripts/MakerSceneManager.cs

```diff
@@ -16,22 +16,21 @@ void start()
 
 	public void uploadToCloud()
 	{
-		/*TODO(hyunSo): works on my laptop but not on my phone. Guess I should have change the camera methad as WebCam
-						also, add rating feature if possible
-		*/
 		camera = Camera.main;
 		if (camera == null) {
 			Debug.Log("Null Camera");
 			return;
 		}
 		width = camera.pixelWidth;
 		height = camera.pixelHeight;
-		Texture2D tex = new Texture2D((int)width, (int)height, TextureFormat.RGB24, false);
+		Texture2D tex = new Texture2D((int)width, (int)width, TextureFormat.RGB24, false);
 		RenderTexture rt = new RenderTexture((int)width, (int)height, 24);
 		camera.targetTexture = rt;
 		camera.Render();
 		RenderTexture.active = rt;
-		tex.ReadPixels(camera.pixelRect, 0, 0);
+		Rect temp = new Rect (0, (height - width)/2, width, width);
```

Comment:
Is this where the cropping takes place?

If so, could you leave some description to explain why you're doing this. It's good (for future readers of the code) to know about the problem being solved.